diff --git a/skia-bindings/build_support/binaries_config.rs b/skia-bindings/build_support/binaries_config.rs
index 3c156b8..76bfcce 100644
--- a/skia-bindings/build_support/binaries_config.rs
+++ b/skia-bindings/build_support/binaries_config.rs
@@ -122,6 +122,10 @@ impl BinariesConfiguration {

         let target = cargo::target();

+        // link the static fontconfig library
+        cargo::add_link_search("/opt/skia-static-libs");
+        cargo::add_static_link_libs(&target, ["fontconfig"]);
+
         cargo::add_static_link_libs(&target, self.built_libraries(true));
         cargo::add_link_libs(&self.link_libraries);
     }
diff --git a/skia-bindings/build_support/platform/linux.rs b/skia-bindings/build_support/platform/linux.rs
index 0678295..0832e12 100644
--- a/skia-bindings/build_support/platform/linux.rs
+++ b/skia-bindings/build_support/platform/linux.rs
@@ -31,9 +31,8 @@ pub fn gn_args(config: &BuildConfiguration, builder: &mut GnArgsBuilder) {
 pub fn link_libraries(features: &Features) -> Vec<String> {
     let mut libs = vec!["stdc++".to_string()];

-    // Use pkg-config for system libraries when available
-    add_pkg_config_libs(&mut libs, "freetype2", &["freetype"]);
-    add_pkg_config_libs(&mut libs, "fontconfig", &["fontconfig"]);
+    // don't add freetype & fontconfig to the list of shared libraries to be linked
+    // since we'll be adding static versions in binaries_config.rs

     if features[feature::GL] {
         if features[feature::EGL] {
diff --git a/skia-bindings/build_support/skia_bindgen.rs b/skia-bindings/build_support/skia_bindgen.rs
index cea5c07..2b446bf 100644
--- a/skia-bindings/build_support/skia_bindgen.rs
+++ b/skia-bindings/build_support/skia_bindgen.rs
@@ -202,6 +202,8 @@ pub fn generate_bindings(
     cargo::rerun_if_file_changed(include_path.join("include"));

     bindgen_args.push(format!("-I{}", include_path.display()));
+    bindgen_args.push("-I/usr/local/include".to_string());
+    cc_args.push("-I/usr/local/include".to_string());
     cc_build.include(include_path);

     for (name, value) in &build.definitions {
